packages/j-db-client/prisma/migrations/20211229072205_first_major_badge_run/migration.sql

```diff
@@ -0,0 +1,36 @@
+-- AlterEnum
+-- This migration adds more than one value to an enum.
+-- With PostgreSQL versions 11 and earlier, this is not possible
+-- in a single migration. This can be worked around by creating
+-- multiple migrations, each migration adding only one value to
+-- the enum.
+
+
+ALTER TYPE "BadgeType" ADD VALUE 'BUG_HUNTER';
+ALTER TYPE "BadgeType" ADD VALUE 'TWENTY_POSTS';
+ALTER TYPE "BadgeType" ADD VALUE 'FIFTY_POSTS';
+ALTER TYPE "BadgeType" ADD VALUE 'SEVENTYFIVE_POSTS';
+ALTER TYPE "BadgeType" ADD VALUE 'ONEHUNDREDFIFTY_POSTS';
+ALTER TYPE "BadgeType" ADD VALUE 'TWOHUNDRED_POSTS';
+ALTER TYPE "BadgeType" ADD VALUE 'TWOHUNDREDFIFTY_POSTS';
+ALTER TYPE "BadgeType" ADD VALUE 'TEN_COMMENTS';
+ALTER TYPE "BadgeType" ADD VALUE 'FIFTY_COMMENTS';
+ALTER TYPE "BadgeType" ADD VALUE 'ONEHUNDRED_COMMENTS';
+ALTER TYPE "BadgeType" ADD VALUE 'TWOHUNDREDFIFTY_COMMENTS';
+ALTER TYPE "BadgeType" ADD VALUE 'FIVEHUNDRED_COMMENTS';
+ALTER TYPE "BadgeType" ADD VALUE 'ONETHOUSAND_COMMENTS';
+ALTER TYPE "BadgeType" ADD VALUE 'CORRECT_TEN_POSTS';
```

Comment:
What's the definition of "correcting a post"? Like leaving either at least one comment or inline comment on it?

Reply:
I'm thinking perhaps leaving an inline comment? Since typically an inline comment is more directly related to a correction, whereas a lot of post comments might be general thoughts/discussion or encouragement ("Nice job!). WDYT?